go/common/identity/identity.go

```diff
@@ -35,6 +37,13 @@ const (
 	tlsKeyFilename  = "tls_identity.pem"
 	tlsCertFilename = "tls_identity_cert.pem"
 
+	tlsEphemeralKeyBaseFilename  = "tls_ephemeral"
+	tlsEphemeralCertBaseFilename = "tls_ephemeral_cert"
```

Comment:
There is no need/reason to also store the certificate as we use public key pinning when verifying certificates, so only the public key really matters. Feel free to remove this part and regenerate the certificate.